src/common/src/lib.rs

```diff
@@ -39,6 +39,7 @@
 #![feature(lint_reasons)]
 #![feature(generators)]
 #![feature(map_try_insert)]
+#![feature(anonymous_lifetime_in_impl_trait)]
```

Comment:
> This feature has no tracking issue, and is therefore likely internal to the compiler, not being intended for general use.

https://doc.rust-lang.org/beta/unstable-book/language-features/anonymous-lifetime-in-impl-trait.html